typedoc.json

```diff
@@ -19,7 +19,10 @@
   },
   "entryPoints": [
     "src/index.ts",
+    "src/lib/mina/account-update.ts",
+    "src/lib/provable/crypto/foreign-curve.ts",
```

Comment:
Foreign curves are already public: https://docs.minaprotocol.com/zkapps/o1js-reference/classes/ForeignCurve

This change makes all the doc comments exposed in the file available for other files to reference in `@link`s.